pkgs/applications/editors/neovim/qt.nix

```diff
@@ -1,5 +1,4 @@
-{ lib, stdenv, mkDerivation, fetchFromGitHub, cmake, doxygen, makeWrapper
-, msgpack, neovim, pythonPackages, qtbase, neovim-qt-unwrapped }:
+{ stdenv, makeWrapper, neovim, neovim-qt-unwrapped }:
```

Comment:
Drive-by cleanup of inputs, not used since 3a25004f2b33b64b9292d923b1a6257b517157a7.